resources/kcp/charts/kyma-environment-broker/values.yaml

```diff
@@ -193,6 +193,13 @@ ems:
 connectivity:
   disabled: true
 
+monitoring:
+  namespace: "kcp-system"
+  chartUrl: "https://storage.googleapis.com/kyma-mps-dev-artifacts/rmi-0.1.0.tgz"
```

Comment:
I think we should remove these `chartUrl` `remoteWriteUrl` default values from OSS KCP code